chart/compass/charts/director/templates/instance-creator-certificate-rotation-job.yaml

```diff
@@ -0,0 +1,153 @@
+                - name: CERT_SVC_OAUTH_CLIENT_CERT
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.name }}
+                      key: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.clientCert }}
+                - name: CERT_SVC_OAUTH_CLIENT_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.name }}
+                      key: {{ .Values.global.externalCertConfiguration.secrets.externalCertSvcSecret.clientKey }}
+                - name: SKIP_SSL_VALIDATION_FLAG
+                  value: {{ .Values.global.instanceCreatorCertConfiguration.secrets.instanceCreatorSvcSecret.skipSSLValidationFlag }}
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ $.Chart.Name }}-{{ .Values.global.instanceCreatorCertConfiguration.rotationCronjob.name }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app: {{ $.Chart.Name }}
+    release: {{ $.Release.Name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ $.Chart.Name }}-{{ .Values.global.instanceCreatorCertConfiguration.rotationCronjob.name }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app: {{ $.Chart.Name }}
+    release: {{ $.Release.Name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+rules:
+  - apiGroups: ["*"]
+    resources: ["secrets"]
+    verbs: ["get", "create", "update", "patch"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ $.Chart.Name }}-{{ .Values.global.instanceCreatorCertConfiguration.rotationCronjob.name }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app: {{ $.Chart.Name }}
+    release: {{ $.Release.Name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ $.Chart.Name }}-{{ .Values.global.instanceCreatorCertConfiguration.rotationCronjob.name }}
+    namespace: {{ $.Release.Namespace }}
+roleRef:
+  kind: Role
+  name: {{ $.Chart.Name }}-{{ .Values.global.instanceCreatorCertConfiguration.rotationCronjob.name }}
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Is this role ever used? I feel like it can be removed

Reply:
I removed it and it still seems to work, so I guess it can be removed.